op03n04b fully annotated

EhsanMoFa opened the pull request:
https://op-musicology.epfl.ch/projects/harmony-annotations/work_packages/912/activity

davor-k commented:
Hi @EhsanMoFa! 

=> 20.2 and 21.2 are actually the same but have two different labels (one is interpreted as iio/v and the other as iv/v)
I would vote for iio6/v on 22.2

=> Also, I would suggest VI(7)/v on 22.1 so that we can see the deceptive resolution V - VI ;)

Let me know your thoughts, thanks! Best

EhsanMoFa commented:
Hey @davor-k 
thanks for the review. It looks good. 
I also found your suggestions good and changed the m.21 (I hope you meant 21:) ) accordingly. Also corrected a small mistake in m.7. 
If you have no more suggestions, I would merge it.
Best
